frontend/src/locales/scripts/en.json5

```diff
@@ -82,10 +82,8 @@
       content: "{openverse} searches across more than 700 million images and audio from open APIs and the {commonCrawl} dataset. It goes beyond simple search to aggregate results across multiple public repositories into a single catalog, and facilitates reuse through features like machine-generated tags and one-click attribution.",
     },
     planning: {
-      content: "Currently {openverse} only searches images and audio, with search for video provided through External Sources. But we plan to add additional media types such as open texts and 3D models, with the ultimate goal of providing access to the estimated 2.5 billion CC licensed and public domain works on the web. All of our code is open source ({frontend}, {api}, {catalog}) and we {community}. You can see what {working}.",
-      frontend: "{openverse} frontend",
-      api: "{openverse} API",
-      catalog: "{openverse} Catalog",
+      content: "Currently {openverse} only searches images and audio, with search for video provided through External Sources. But we plan to add additional media types such as open texts and 3D models, with the ultimate goal of providing access to the estimated 2.5 billion CC licensed and public domain works on the web. All of our code is open source and can be accessed at the {repository}. We {community}. You can see what {working}.",
+      repository: "{openverse} GitHub repository",
```

Comment:
Since `repository` is a new entry in the i18n text, a corresponding entry needs to be added to the [i18n strings for testing](https://github.com/WordPress/openverse/tree/6f5a10b30f87af2e3d4fd3a6bfe32eba3dd21ea9/frontend/test/locales).

Also I'm not entirely sure but @WordPress/openverse-frontend can confirm if we need to prevent "GitHub" from being translated as that's the name of the company, like we do for Openverse?

Reply:
I agree with excluding `GitHub` from translations, @dhruvkb.